schema.json

```diff
@@ -113,6 +114,7 @@
           "type": "string",
           "title": "Update-Path Schema",
           "description": "Folder where the application expects auto-updated OTA packages to be found. Default is value specified in 'target path'.",
+          "default": "",

```

Comment:
Is there any json-schema way to actually tie this value to target-path as the `description` implies?

Reply:
I am setting the value for each of these paths using the JsonHandler class I created.
        These are the relevant calls in ApplyState.
```Java
        _targetPath = new File(JsonHandler.getOptString("global", "target-path"));
        _installPath = new File(JsonHandler.getOptString("global", "install-path"));
        _updatePath = new File(JsonHandler.getOptString("global", "update-path"));
```

Reply:
Updated these descriptions in [66ee14f](https://github.com/datalogic/scan2deploy-android-schema/pull/1/commits/66ee14fd43b7b5908d255e246018270e8b5f3bf9).

Reply:
After IRL discussion, I have reverted some of these changes and added a definitions section to the schema allowing target-path, install-path, and update-path to all inherit from a base-path value.